pulumi/python_integration_tests/__main__.py

```diff
@@ -70,10 +79,15 @@ def main() -> None:
         # on-disk Grapl repo.
         # FIXME: make python integration tests work in AWS.
 
+        container_images = (_python_integration_container_images(artifacts),)
```

Comment:
uhhhhh you sure about this? this makes container_images a tuple

Reply:
Ugh, no. I must have typed a paren by accident or soemthing here and my IDE autoexpanded. Thanks for catching this